zebra/rt_netlink.c

```diff
@@ -1462,9 +1476,9 @@ static void _netlink_route_debug(int cmd, const struct prefix *p,
 	if (IS_ZEBRA_DEBUG_KERNEL) {
 		char buf[PREFIX_STRLEN];
 		zlog_debug(
-			"netlink_route_multipath(): %s %s vrf %u(%u)",
+			"netlink_route_multipath(): %s %s vrf %s(%u) table: %u",
```

Comment:
table_id maybe?

Reply:
It is true, that table_id is more clear. Done.